sql/src/main/java/io/crate/operation/reference/sys/node/fs/NodeStatsFsDataExpression.java

```diff
@@ -43,10 +43,12 @@ public NodeStatsFsDataExpression() {
 
     @Override
     protected Map<String, Object> valueForItem(final ExtendedFsStats.Info input) {
-        return new HashMap<String, Object>() {{
-            put(NodeFsStatsExpression.DEV, input.dev());
-            put(NodeFsStatsExpression.PATH, input.path());
-        }};
+        return new HashMap<String, Object>() {
+            {
+                put(NodeFsStatsExpression.DEV, input.dev());
```

Comment:
Not relevant for this PR, but I think we should change this here to not create anynomous HashMap classes

Reply:
@mxm check also here: https://blog.jooq.org/2014/12/08/dont-be-clever-the-double-curly-braces-anti-pattern/